cloud/google/machineactuator.go

```diff
@@ -158,13 +155,7 @@ func (gce *GCEClient) CreateMachineController(cluster *clusterv1.Cluster, initia
 		return err
 	}
 
-	// Create the configmap so the machine setup configs can be mounted into the node.
-	// TODO: create the configmap during bootstrapping instead of being buried in the machine actuator code.
-	machineSetupConfigs, err := gce.configWatch.ValidConfigs()
-	if err != nil {
-		return err
-	}
-	yaml, err := machineSetupConfigs.GetYaml()
+	yaml, err := gce.machineSetupConfig.GetYaml()
```

Comment:
Do we need a nil check before here too since we are trying to use the machineSetupConfig?